src/test/scala/com/microsoft/hyperspace/SampleData.scala

```diff
@@ -25,10 +27,43 @@ object SampleData {
     ("2017-09-03", "fd093f8a05604515957083e70cb3dceb", "facebook", 1, 3000),
     ("2017-09-03", "af3ed6a197a8447cba8bc8ea21fad208", "facebook", 1, 3000),
     ("2017-09-03", "975134eca06c4711a0406d0464cbe7d6", "facebook", 1, 4000),
-    ("2017-09-03", "e90a6028e15b4f4593eef557daf5166d", "zanahoria", 2, 3000),
-    ("2017-09-03", "576ed96b0d5340aa98a47de15c9f87ce", "facebook", 2, 3000),
-    ("2017-09-03", "50d690516ca641438166049a6303650c", "ibraco", 2, 1000),
-    ("2017-09-03", "380786e6495d4cd8a5dd4cc8d3d12917", "facebook", 2, 3000),
-    ("2017-09-03", "ff60e4838b92421eafc3e6ee59a9e9f1", "mi perro", 2, 2000),
-    ("2017-09-03", "187696fe0a6a40cc9516bc6e47c70bc1", "multicines", 4, 3000))
+    ("2018-09-03", "e90a6028e15b4f4593eef557daf5166d", "ibraco", 2, 3000),
+    ("2018-09-03", "576ed96b0d5340aa98a47de15c9f87ce", "facebook", 2, 3000),
+    ("2018-09-03", "50d690516ca641438166049a6303650c", "ibraco", 2, 1000),
+    ("2019-10-03", "380786e6495d4cd8a5dd4cc8d3d12917", "facebook", 2, 3000),
+    ("2019-10-03", "ff60e4838b92421eafc3e6ee59a9e9f1", "mi perro", 2, 2000),
+    ("2019-10-03", "187696fe0a6a40cc9516bc6e47c70bc1", "facebook", 4, 3000))
+
+  // column names and partition keys for partitioned case
+  val colnames = Seq("Date", "RGUID", "Query", "imprs", "clicks")
+  val partitionKey1 = "Date"
+  val partitionKey2 = "Query"
+
+  def saveTestDataNonPartitioned(spark: SparkSession, path: String, colNames: String*): Unit = {
+    import spark.implicits._
+    assert(colNames.length == 5)
```

Comment:
why do you limit this?

Reply:
I wanted to make sure passed columns count matches with the DF's column count. I checked the `toDF()` call and realized it already has `require(schema.size == colNames.size`. So I removed my assert.